From 55e8dc781db80d22fc317f1a9671983e9abc5ec3 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Fri, 27 Mar 2020 07:15:55 +0000
Subject: [PATCH] examples/pxScene2d: Include <functional> header

This fixes
 src/pxVideo.cpp:25:22: error: 'placeholders' is not a namespace-name

with gcc9 this is explicit include needed now

Source: COMCAST
License: Apache-2.0

Upstream-Status: Pending

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 examples/pxScene2d/src/pxVideo.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/examples/pxScene2d/src/pxVideo.cpp b/examples/pxScene2d/src/pxVideo.cpp
index 9dc50634f..42e578793 100644
--- a/examples/pxScene2d/src/pxVideo.cpp
+++ b/examples/pxScene2d/src/pxVideo.cpp
@@ -18,6 +18,7 @@
 
 // pxText.h
 
+#include <functional>
 #include <iostream>
 #include <string>
 #include "pxVideo.h"
-- 
2.17.1

